src/elexmodel/handlers/data/CombinedData.py

```diff
@@ -48,6 +48,7 @@ def __init__(
             data.update(data[result_cols].fillna(value=0))
             data.loc[indices_with_null_val, "percent_expected_vote"] = 0
 
+        self.n_minimum_for_outlier_detection_model = 20
```

Comment:
Doing this makes sense, but how did you come up with 20? 🤔

Reply:
entirely arbitrary, but purposefully set higher than the 10 we need to run the model